src/Test.UnitTests.Sarif/Test.UnitTests.Sarif.csproj

```diff
@@ -23,6 +23,7 @@
     <None Remove="TestData\PrereleaseCompatibilityTransformer\ExpectedOutputs\ArtifactsWithRoles.sarif" />
     <None Remove="TestData\PrereleaseCompatibilityTransformer\ExpectedOutputs\ComprehensiveToolProperties.01-24.sarif" />
     <None Remove="TestData\PrereleaseCompatibilityTransformer\ExpectedOutputs\ComprehensiveToolProperties.sarif" />
+    <None Remove="TestData\PrereleaseCompatibilityTransformer\ExpectedOutputs\DateTimeInPropertyBag.sarif" />
```

Comment:
>DateTimeInPropertyBag.sarif [](start = 78, length = 27)

A file per bug fix is too granular. Can we please think about collapsing these tests into a smaller set? For example, a file that was dedicated to serializing all types. Or a file that's dedicated to some higher level class.

Generally, for file-based data-driven tests, the goal is to have the smallest set of coherent driver files. Some patterns are mutually exclusive and require multiple files.

Reply:
Fixed, see my reply to your other comment on this issue.

---
In reply to: [316736735](https://github.com/microsoft/sarif-sdk/pull/1651#discussion_r316736735) [](ancestors = 316736735)